edge-modules/TestAnalyzer/swagger.yaml

```diff
@@ -127,19 +106,21 @@ definitions:
       lastReceivedAt:
         type: "string"
         format: date-time
-  ResponseStatus:
+  TestOperationResult:
     type: "object"
     required:
-    - "moduleId"
-    - "statusCode"
-    - "enqueuedDateTime"
+    - "source"
+    - "result"
+    - "resultType"
+    - "createdOn"
     properties:
-      moduleId:
+      source:
         type: "string"
-      statusCode:
+      result:
+        type: "string"
+      resultType:
```

Comment:
There is a way to make this an enum: https://swagger.io/docs/specification/data-models/enums/

I'm not sure if it is worth doing since the type would remain as string.

Reply:
I changed back to use string for test result type in TestOperationResult class; which don't need to re-generate AnalyzerClient when trying to support a new result type.

Reply:
@and-rewsmith, I don't need to re-generate AnalyzerClient since swagger.yaml is not changed. right?

Reply:
re-generated since I rename ResultType to Type in TestOperationResult.